### ISSUE TITLE
Help in Understanding [oversim/src/underlay/inetunderlay/InetUnderlay.ned]

### ISSUE THREAD

**Issue by sairamch04** [aarizaq/Oversim-20121206#3]
what do **overlayBackboneRouter**, **overlayBackboneRouter**, **accessRouter**, **backboneRouter**, **outRouter **do . Are they required for Mobile Adhoc Network ?

**Comment by aarizaq** [comment]
Basically, you only need to add a wireless interface and include a manet routing protocol, similar to the inet nodes.

Date: Tue, 16 Feb 2016 02:11:12 -0800

**Comment by aarizaq** [comment]
No, you don't need these nodes for an ad-hoc network, the nodes in a ad-hoc networks have router functionalities.


Date: Tue, 16 Feb 2016 02:11:12 -0800

**Comment by sairamch04** [comment]
Thanks for the reply. i also added **HostAutoConfigurator** to assign IP address to wlan interface.
